test/e2e_node/builder/build.go

```diff
@@ -35,6 +35,7 @@ var buildTargets = []string{
 	"test/e2e_node/e2e_node.test",
 	"vendor/github.com/onsi/ginkgo/ginkgo",
 	"cluster/gce/gci/mounter",
+	"cluster/gce/gci/credential-provider",
```

Comment:
we should be moving things out of the cluster dir, not adding new things and referencing them there...